### PR TITLE
opencv/maskrcnn: Added maskR-CNN example in Cpp

### PR DESCRIPTION
**Pull request by grigorovici-andreea**
Added maskR-CNN example in Cpp using OpenCV
Added documentation and output example.

Solves: #306 

Signed-off-by: Andreea Grigorovici <andreea.grigorovici@analog.com>

### CODE REVIEW

File: bindings/opencv/maskrcnn/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(aditof-opencv-maskrcnn)
+
+add_executable(${PROJECT_NAME} main.cpp)
+
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/frozen_inference_graph.pb
```
Comment (DanielGuramulta) [inline bindings/opencv/maskrcnn/CMakeLists.txt:7]: Can't we just give the user some documentation on how to get this files? They seem a bit big to be included in the repo and could prolong the duration of `cloning` the repo even when this example is not needed. (frozen_inference_graph.pb is 64MB)

File: bindings/opencv/CMakeLists.txt [hunk 2]
```diff
@@ -29,7 +29,12 @@ if(OPENCV_FOUND OR
     else()
         message(STATUS "The dnn example will be built (OpenCV version >= 3.4.1)")
         add_subdirectory(dnn)
-
+        if(OpenCV_VERSION VERSION_LESS "3.4.3")
+            message(WARNING "The maskR-CNN example is only available for OpenCV version >= 3.4.3! This example won't be built!")
+        else()
+            message(STATUS "The maskR-CNN example will be built (OpenCV version >= 3.4.3)")
```
Comment (dNechita) [inline bindings/opencv/CMakeLists.txt:35]: I don't think it's necessary to print this.

File: bindings/opencv/maskrcnn/CMakeLists.txt (hunk 1 above)
Comment (dNechita) [reply bindings/opencv/maskrcnn/CMakeLists.txt:7]: If the files are binary and/or have large sizes, then we shouldn't include them in the repository.

File: bindings/opencv/CMakeLists.txt (hunk 2 above)
Comment (grigorovici-andreea) [reply bindings/opencv/CMakeLists.txt:35]: For DNN example both of these messages are printed also. I think this should be consistent. Either I delete from both, either I let it this way. What do you think?

Comment (dNechita) [reply bindings/opencv/CMakeLists.txt:35]: delete both

File: bindings/opencv/maskrcnn/CMakeLists.txt (hunk 1 above)
Comment (grigorovici-andreea) [reply bindings/opencv/maskrcnn/CMakeLists.txt]: solved.

File: bindings/opencv/CMakeLists.txt (hunk 2 above)
Comment (grigorovici-andreea) [reply bindings/opencv/CMakeLists.txt]: solved.

**Pull request merged**